### DISCUSSION TITLE
Project Dead?

### DISCUSSION THREAD

**Discussion by antl3x** [proofrock/duckrg#3] [General]
Hey, is the project dead? Didn't found any release/executable.

**Comment by proofrock** [comment]
Hi @antl3x ! Thanks for writing.

I... am not sure 😅 I mean: this was most of all a proof of concept, and since then I reverted back to focus on the upstream project, `ws4sqlite`. 

Please see [this discussion](https://github.com/proofrock/ws4sqlite/discussions/44), I have plans (if there will be interest) to support different databases, including duckdb; the main issue is that the [go "driver"](github.com/marcboeker/go-duckdb) uses gcc and it's quite difficult to cross-compile.

Anyway. In the next days I will update and compile this project, I didn't really notice that there wasn't a release; so, if you are interested you can test. Thanks!

**Comment by antl3x** [comment]
Thanks for the detailed answer!

What differences between the project and https://github.com/goblgobl/duckdb-proxy would you highlight

I will check the release when it's ready.

**Reply by proofrock** [reply]
Agh, I just wrote a long answer, which I deleted 😐 Oh well it happens.

I released a "poc" version. Take a look if it's useful to you; I unfortunately don't have a cross-compilation process ready, so it's just linux/amd64, let me know if you need something else.

About the project you mention: interesting! Evolutionary convergence :-) At a first glance, I'd say that `ws4sqlite` and family ("we") support multiple databases and multiple statements per request (executed in transactions), which is more useful... at a price (e.g. we use config files). For the rest, we have a little more features, like init statements, or stored statements referrable by name; "they" have some configs that we prefer to allow in the sql (e.g. limit number of returned rows)... it's all a matter of choices.

Please read about the features in the [sumup](https://docs.sqliterg.dev/features) (of course, replace `sqlite` with `duckdb` 😁), and you may be interested in the [security](https://docs.sqliterg.dev/security) ones too.

Thanks!
